src/test/shared/logger/sharedFileTransport.test.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import vscode from 'vscode'
+import assert from 'assert'
+import { SharedFileTransport, flushIntervalMillis } from '../../../shared/logger/sharedFileTransport'
+import { FileSystemCommon } from '../../../srcShared/fs'
+import { stub, SinonStub } from 'sinon'
+import { MESSAGE } from '../../../shared/logger/debugConsoleTransport'
```

Comment:
off-topic: I wonder if there's a way we can hide this `MESSAGE` symbol from callers, it's a strange interface.

Reply:
The `MESSAGE` symbol really confused me at first.

This doc says that 2 separate symbols, even if created with the same string cannot be equal: [see the 2nd code snippet](https://www.typescriptlang.org/docs/handbook/symbols.html). But then it works when the winston backend injects their own property with their [own `MESSAGE` symbol](https://github.com/winstonjs/triple-beam/blob/master/index.js#L23).

I'm not sure if we can change, seems like a built in thing: 
"[Symbol.for('message'): complete string message set by "finalizing formats"](https://github.com/winstonjs/winston#streams-objectmode-and-info-objects)"